src/eventDispatchers/mouseToolEventDispatcher.js

```diff
@@ -30,6 +31,7 @@ const enable = function (element) {
   element.addEventListener(EVENTS.MOUSE_MOVE, mouseMove);
   element.addEventListener(EVENTS.MOUSE_UP, mouseUp);
   element.addEventListener(EVENTS.MOUSE_WHEEL, mouseWheel);
+  element.addEventListener(EVENTS.KEY_DOWN, keyDown);
```

Comment:
We have a lot of `misc` listeners like this popping up (IE. some of the `cornerstone` specific ones that aren't tied to touch/mouse). I'm still noodling on the best place for them to live.

Reply:
I agree there must be a neater way to store the keyboard events. Both mouseTools  and keyboardTools can utilize them, so they are a bit in limbo. The solution will likely become clearer when we think about putting keyboard only tools back in v3, are there any on our current roadmap?

Reply:
The only one I'm aware of is `keyboardStackScroll`. I "think" it's on @galelis's hit list. This is one of those discussion items where we're not sure if a `keyboard` tool is really just another `interactionParadigm` for a mouse/touch tool. IE.

`wwwcKeyboardTool` could be the `wwwcTool` with keybindings that apply a strategy. It's a question of: what's most intuitive? how would this impact instantiating and activating tools?